index.bs

```diff
@@ -1529,6 +1530,9 @@ To <dfn>deserialize local value</dfn> given |local protocol value|, |realm| and
 1. If |local protocol value| matches the [=PrimitiveProtocolValue=] production, return
    [=deserialize primitive protocol value=] with |local protocol value|.
 
+1. If |local protocol value| matches the <code>script.Channel</code> production,
```

Comment:
Can we pick other naming, as `channel` [is already used](https://github.com/GoogleChromeLabs/chromium-bidi#field-channel) in out implementation-specific extension for other purpose?